src/main/context-handler.ts

```diff
@@ -48,13 +48,7 @@ export class ContextHandler {
 
   public setupPrometheus(preferences: ClusterPrometheusPreferences = {}) {
     this.prometheusProvider = preferences.prometheusProvider?.type;
-    this.prometheusPath = null;
-
-    if (preferences.prometheus) {
-      const { namespace, service, port } = preferences.prometheus;
-
-      this.prometheusPath = `${namespace}/services/${service}:${port}`;
-    }
+    this.preferences = preferences;
```

Comment:
Final comment, I don't think that we need to save the whole of this here, just like we don't for `this.prometheusProvider`.

Reply:
What do you think from this variant?